doc/source/man.rst

```diff
@@ -32,206 +32,217 @@ rho performs four major tasks:
+  Sets the host name, IP address, or IP address range to use when running a scan. You can also provide a path and file name for a file that contains a list of hosts or IP addresses or ranges, where each item is on a separate line. The following examples show several different formats that are allowed as values for the ``--hosts`` option:
 
-  1. A specific hostname:
+  * A specific host name:
 
     --hosts server.example.com
 
-  2. A specific IP address:
+  * A specific IP address:
 
-    --hosts 1.2.3.4
+    --hosts 192.0.2.19
 
-  3. An IP address range:
+  * An IP address range:
 
-    --hosts "1.2.3.[4:255]"
+    --hosts 192.0.2.[1:254]
+
+  * A file:
+
+    --hosts /home/user1/hosts_file
 
 ``--auth auth_profile``
 
-  This contains the name of the authentication profile (created with ``rho auth add``) to use to authentication to the servers being scanned. To add more than one auth profile to the network profile provide a list separated with a space. For example:
+  Contains the name of the authentication profile to use to authenticate to the systems that are being scanned. If the individual systems that are being scanned each require different authentication credentials, you can use more than one authentication profile. To add multiple authentication profiles to the network profile, separate each value with a space, for example:
 
   ``--auth first_auth second_auth``
 
-  IMPORTANT: This auth profile must exist before attempting to add the authentication profile to the network profile.
+  **IMPORTANT:** An authentication profile must exist before you attempt to use it in a network profile.
 
 ``--sshport=ssh_port``
 
-  This value is used to support discovery on a non-standard port. Discovery takes place by default on port 22.
+  Sets a port to be used for the scan. This value supports discovery on a non-standard port. By default, the scan runs on port 22.
 
 ``--vault=vault_file``
 
-  This contains the path and filename of the file containing the vault password. If this option is used the file should be limited in access on the system.
+  Contains the path and file name of the file that contains the vault password. Because the encrypted Rho data could contain sensitive information, make sure that this vault password file is stored in a location that has limited access.
+
+The information in a network profile might change as the structure of the network changes. Use the ``rho profile edit`` command to edit a network profile to accommodate those changes.
+
+Although ``rho profile`` options can accept more than one value, the ``rho profile edit`` command is not additive. If you edit a network profile and pass a new value for an option, the command overwrites all current values for that option. To add additional values to an option, but keep the current values, you must include the option with both its current and new values in the ``rho profile edit`` command.
```

Comment:
I don't think this is correct - if the user doesn't pass an option to ``rho profile edit``, we keep the old value. I see that the previous version of the man page also had the bad information, so it's not your draft, it was just wrong before.

Reply:
The edit will find the entry by looking by name and then any additional option provided will be updated. If some information is already set but the option to edit it is not provided it will remain the same.

Reply:
Clarified with Elyezer (you were offline) that you do still have to enter the current and new values for the option that you want to change. 

Change made:
Although ``rho profile`` options can accept more than one value, the ``rho profile edit`` command is not additive. To edit a network profile and add a new value for an option, you must enter both the current and the new values for that option. If you do not include an option in the ``rho profile edit`` command, then the value for that option is not changed.

Reply:
Actually, I changed my mind. Change made:

Although ``rho profile`` options can accept more than one value, the ``rho profile edit`` command is not additive. To edit a network profile and add a new value for an option, you must enter both the current and the new values for that option. Include only the options that you want to change in the ``rho profile edit`` command. Options that are not included are not changed.